src/collections/_documentation/workflow/releases/health.md

```diff
@@ -41,6 +41,26 @@ The release details page focuses on an individual release. Elements of the relea
 
 [{% asset releases/release_details.png alt="View of the release details page showing an overview graph, files changed, and other details." %}]({% asset releases/release_details.png @path %})
 
+Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy.
+
+Crashed 
+
+: The app had an explicit hard crash. You’ll likely see a corresponding Sentry issue for the crash.
```

Comment:
Please check with @marandaneto whether we should use the term "unhandled error" rather than crash in the description.

Reply:
I'd go with unhandled error as well because of https://develop.sentry.dev/sdk/event-payloads/exception/#attributes-1

for the mobile world, the term hard crash is known, but in JS this might be confusing, as we use the `mechanism.handled` to tell either if it's a hard crash or not, I'd stick to the term handled/unhandled for keeping it semantic.

Reply:
This depends on how we're planning to implement this in other SDKs. For example, will the Python SDK set the session to "crashed", if the web server continues to run, but there was an unhandled exception in the web request? I'd say, probably not.

This is an excellent point to think about. For now, I'd stick with the current description, however.